src/cdk/scrolling/fixed-size-virtual-scroll.ts

```diff
@@ -174,7 +174,8 @@ export class FixedSizeVirtualScrollStrategy implements VirtualScrollStrategy {
     }
 
     this._viewport.setRenderedRange(newRange);
-    this._viewport.setRenderedContentOffset(this._itemSize * newRange.start);
+    const newRangeStart = this._viewport.getRenderedRange().start * this._itemSize;
```

Comment:
This change doesn't seem quite right to me. I notice that `setRenderedRange` is special-cased for append only mode, I think `setRenderedContentOffset` may need something similar.

Reply:
I get the point, it's the responsability of the viewport to do this logic, otherwise it would be in all classes implementing VirtualScrollStrategy